config_sample.yml

```diff
@@ -2,11 +2,11 @@
 #Copy its content to config.yml file
 #It is used only when you run tests using IDE
 
-browser: FF #acceptable values: FF, CHROME, GHOST, ANDROID
+browser: CHROME #acceptable values: FF, CHROME, GHOST, ANDROID
```

Comment:
What about those fancy `CHROMEMOBILE` and `CHROMEMOBILEMERCURY` mentioned in the README? Is it something different? If not, I'd like them to be mentioned here in the comment as well ;)